src/fe-small-advertisements/src/reducers/index.ts

```diff
@@ -0,0 +1,7 @@
+import personReducer from '../features/person/personSlice';
+
+export const rootReducer = {
+  person: personReducer,
+};
+
+export type RootState = typeof rootReducer;
```

Comment:
If you look closely to other example you might find out that in domain driven file structure usually dont have a `reducers` folder either. You might deleting it from here as well please?

Reply:
I deleted it and tried to do a DDD with what we have.